src/js/components/leftPanel/layersPanel/GenericLayerControl.tsx

```diff
@@ -36,6 +36,23 @@ const GenericLayerControl = (props: LayerControlProps): React.ReactElement => {
     }
   };
 
+  const returnSubtitle = (): JSX.Element | undefined => {
+    let subTitle = '';
+    if (layer?.metadata?.metadata) {
+      subTitle = (layer?.metadata?.metadata as any).subtitle;
+      return (
+        <>
+          <br />
+          <span className="layer-subtitle">{subTitle}</span>
+        </>
+      );
```

Comment:
On each GFW API layer in addition to metadata they have sublabel property with supported languages, we should try to pull from that first and match the language, if all fails we can try and extract subtitle property from metadata.

![image](https://user-images.githubusercontent.com/12076555/77682367-36d0e680-6f6d-11ea-9ea5-37c9bbb84230.png)

This sublabel object may need to be extracted at the place we extract all this info in mapController. line 200 or about there.